test/include/userobjects/CheckGhostedBoundaries.h

```diff
@@ -0,0 +1,32 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "GeneralUserObject.h"
+
+/**
+ * A native way to check if all boundaries are gathered to every single processor.
```

Comment:
Do you mean "native" or do you mean "naive"?

Reply:
Yes.

Reply:
I will change ..